src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
   color: var(--theme-tab-active-color);
 }
 
 .tab.deselected {
-  background: var(--theme-tab-inactive-background);
-  box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
-  color: var(--theme-tab-inactive-color);
+  /* react-css-modules generateAppendClassName requires that this exist */
 }
 
 .tab.initial {
   width: var(--theme-tab-initial-width);
+  height: var(--theme-tab-initial-height);
+  background-position: var(--theme-tab-inactive-background-position);
   opacity: var(--theme-tab-initial-opacity);
 }
 
 .tab.open {
-  width: var(--theme-tab-open-width);
+  height: var(--theme-tab-open-height);
   opacity: var(--theme-tab-open-opacity);
 }
 
 .tab.closed {
   width: var(--theme-tab-closed-width);
   opacity: var(--theme-tab-closed-opacity);
+  transition: width 0.5s var(--animation-curve);
+}
+
+.tab.selected-closed {
+  width: var(--theme-tab-closed-width);
+  height: var(--theme-tab-closed-height);
+  background-position: var(--theme-tab-active-background-position);
```

Comment:
I didn't notice the hover effect before, but you're right, if I chase the tab with my mouse as it shrinks then the background will flicker.